SafeExamBrowser.SystemComponents/VirtualMachineDetector.cs

```diff
@@ -33,6 +37,17 @@ public bool IsVirtualMachine()
 			isVirtualMachine |= manufacturer.Contains("parallels software");
 			isVirtualMachine |= model.Contains("virtualbox");
 			isVirtualMachine |= manufacturer.Contains("qemu");
+			
+			var macAddr = (from nic in NetworkInterface.GetAllNetworkInterfaces() where nic.OperationalStatus == OperationalStatus.Up select nic.GetPhysicalAddress().ToString()).FirstOrDefault();
+			isVirtualMachine |= ((byte.Parse(macAddr[1].ToString(), NumberStyles.HexNumber) & 2) == 2 || macAddr.StartsWith("080027"));
+
+			using (var searcher = new ManagementObjectSearcher("root\\CIMV2", "SELECT DeviceID FROM Win32_PnPEntity"))
+
+				foreach (ManagementObject queryObj in searcher.Get())
```

Comment:
ManagementObject is also disposable, thus please encapsulate it in a using statement.